stock_picking_unbuild/models/stock_picking.py

```diff
@@ -0,0 +1,41 @@
+from odoo import _, api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    is_display_unbuild = fields.Boolean(compute="_compute_display_unbuild", store=True)
```

Comment:
Isn't store=True already the default value or is is_display_unbuild normally created with store=False?

Reply:
> Isn't store=True already the default value or is is_display_unbuild normally created with store=False?

When we use compute attribute of the field so the field does not store the value in the database, we have to add store=True for storing the value.